lnbits/db.py

```diff
@@ -149,14 +156,7 @@ def _parse_timestamp(value, _):
 
             psycopg2.extensions.register_type(
                 psycopg2.extensions.new_type(
-                    (1184, 1114),
-                    "TIMESTAMP2INT",
-                    _parse_timestamp
-                    # lambda value, curs: time.mktime(
-                    #     datetime.datetime.strptime(
-                    #         value, "%Y-%m-%d %H:%M:%S.%f"
-                    #     ).timetuple()
-                    # ),
+                    (1184, 1114), "TIMESTAMP2INT", _parse_timestamp
```

Comment:
Regarding:
```py
def _parse_timestamp(value, _):
    f = "%Y-%m-%d %H:%M:%S.%f"
    if not "." in value:
        f = "%Y-%m-%d %H:%M:%S"
    return time.mktime(datetime.datetime.strptime(value, f).timetuple())
```
If `expiry` is `None` then it throws:
```py
lnbits/lnbits/db.py", line 136, in _parse_timestamp
    if not "." in value:
TypeError: argument of type 'NoneType' is not iterable
````

Reply:
Is a typing hint enough?